_source/_posts/2022-04-05-state-of-ffi-java.md

```diff
@@ -0,0 +1,280 @@
+---
+layout: blog_post
+title: "Does Java finally have a secure alternative to JNI?"
```

Comment:
What are the keywords for this post and are they included in the title, headings, and URL? 

Keywords aren't required but should be considered if you're optimizing for SEO.